src/run.js

```diff
@@ -176,8 +176,10 @@ const processPage = ({
       page.on('response', response => {
         const responseUrl = response.url()
         const ct = response.headers()['content-type'] || ''
-        if (!response.ok()) {
-          return safeReject(new Error(`${response.status()} on ${responseUrl}`))
+        if (response.status() >= 400) {
+          return safeReject(
+            new Error(`Response status ${response.status()} on ${responseUrl}`)
+          )
```

Comment:
This makes me nervous. I need to try to understand how this works. 

If you have a page like:
```html
<link rel="stylesheet" href="http://httpbin.org/redirect-to?url=/real.css">
```
then, *that* URL won't have a `response.text()` because it's a 30X redirect. But the `stylesheetAsts` objects needs the original URL. 
What you want is to accept it (i.e. not using `response.ok()`) but we parse it, we need to do the parsing based on the css it redirects to. 

To remedy this, I think I'd like to write a unit test to simulate it. But there's so much going on in https://github.com/peterbe/minimalcss/pull/88 that perhaps it's best to wait.